### DISCUSSION TITLE
TRIP CALCULATOR : What calculations should be made in the trip calculator?

### DISCUSSION THREAD

**Discussion by Omonotuaga** [PranishKansakar/Tafesa240154#6] [Ideas]
Good day my lovelies, What calculations should be made in the trip calculator?

**Comment by PranishKansakar** [comment]
In calculating the amount owed by a customer after returning a car,  consider the starting and ending kilometers, along with the number of days hired and the price per day.

Use the formula, amount = days hired * price per day, efficiently determines the customer's payment, ensuring a fair transaction.
